features/admin.branding.v1/components/screen-variation-dropdown.tsx

```diff
@@ -0,0 +1,126 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
```

Comment:
Hide the drop down if there are no variations specified.

Reply:
or if the variation has only one option.

Reply:
addressed with https://github.com/wso2/identity-apps/pull/6027/commits/f4caedf5091ff4248069acafa3599ddc0f886d53